views/index.view.php

```diff
@@ -0,0 +1,19 @@
+<?php include 'partials/head.php'; ?>
+  <div class="row">
+    <div class="col">
+      <table>
+      <?php
+      foreach($users as $user):
```

Comment:
This looks like a great opportunity to talk about data driven templating in php. This might even be a good jumping off point for the more focused discussions that we talked about. I'm a bit torn though between whether it would be better to start our focus on the UI, which IMO is easier to understand, or the DB, where I feel like we have more interest as a group. @code-mates/codemates